config/ecs/container-insights/otel-task-metrics-config.yaml

```diff
@@ -58,9 +73,10 @@ processors:
       - metric_name: ecs.task.storage.write_bytes
         action: update
         new_name: StorageWriteBytes
+
   resource:
     attributes:
-      - key: ClusterName
+      - key: Cluster
```

Comment:
We should not change this dimension. I meant to add only the new dimensions. Existing dimensions should remain same.

Reply:
I thought we were going to use the original name which our customer used to see.

Reply:
Yea, customer see `ClusterName` and `ContaienrName` in current Performance Log event. https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-reference-performance-logs-ECS.html

Reply:
So for the metric name we have in insights we refer https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-reference-performance-logs-ECS.html
But for the name we don't have for the insights, we refer this one: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint-v4.html

Reply:
Yes, thats the plan. To finalize, I sent and email to Container Insights PM and our PM (including you). Let's wait for their response.